scripts/run-binaries-all.sh

```diff
@@ -0,0 +1,3 @@
+#! /bin/sh
+
+ansible-playbook -b -f 20 ydb_platform.ydb.binaries_all
```

Comment:
Why do you recommend these? I always run just `ansible-playbook <playbook-name>`, and it works fine. And for advanced people running this at a larger-than-default scale, it should be trivial to figure out `-f`.